mailpoet/tests/integration/Segments/DynamicSegments/Filters/WooCommerceCategoryTest.php

```diff
@@ -34,7 +36,6 @@ public function _before(): void {
     $this->wooCommerceCategory = $this->diContainer->get(WooCommerceCategory::class);
     $this->subscribersRepository = $this->diContainer->get(SubscribersRepository::class);
 
-    Database::loadSQL('createWCLookupTables');
```

Comment:
This was the last use of the method Database::loadSQL(). Do you reckon we should remove it? I'm not sure as it could be useful in the future, but I'm inclined to say we should as we are not currently using it.

Reply:
I was thinking that it could be useful in the future. But you're right we don't use it so let's remove it.